package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Volkswagen detects when your tests are being run in a CI server, and makes them pass.",
   "main": "./.these/aren't/the/source/files/you're/looking/for.js",
   "dependencies": {
+    "chai": "^3.3.0",
```

Comment:
It would be. Dependencies defined here would be delivered to users installing `volkswagen` package. Just trust me. It MUST be in devDependencies.
http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies